playground/layouts/error.vue

```diff
@@ -6,6 +6,8 @@
 
 <script>
 export default {
-  props: ['error']
+  props: {
+    error: Error
```

Comment:
This prop is not always an Error.

See for example this typing in Nuxt 3:

https://github.com/nuxt/framework/blob/888bd7c145cf276bfbb93e9610ba3b6c8786007f/packages/nuxt/src/app/nuxt.ts#L81-L88

Reply:
Would it be fair to copy that typing? As it cannot be referenced, I think, as a property of an unexported interface.

Reply:
Yes. The source does not come from that repo but from the nitro renderer within Bridge.

Reply:
Wait. I just twigged this is the _playground_. Scratch that. Your original solution is fine! 😆